lib/daisy_ui_components/card.ex

```diff
@@ -11,8 +11,43 @@ defmodule DaisyUIComponents.Card do
   @paddings ~w(normal compact)
   @placements ~w(top bottom)
 
+  @doc """
+
+  Returns the modifiers associated with the given reference.
+
+  ## Parameters
+
+  - reference - a unique identifier to fetch the relevant modifiers.
+  ## Description
+   Retrieves the modifiers linked to the specified reference.
+
+  """
   def modifiers, do: @modifiers
```

Comment:
I am pretty sure you want this to be public but not documented.

maybe `doc false` is the way

Reply:
yep, I need to add doc false on these functions